test/util.spec.js

```diff
@@ -160,7 +160,9 @@ describe('.util', () => {
         .then(out => expectTimeout(ipfs.object.get(out[0].hash), 4000))
     })
 
-    it('with wrap-with-directory=true', (done) => {
+    it('with wrap-with-directory=true', function (done) {
```

Comment:
I think the `function` keyword is a leftover here?

Reply:
I'm using `function` here because on the next time we want to call `this.timeout`. Mocha provides an object with this method on it for the function context (`this`) but arrow functions do not allow you to change the context.

Reply:
Got it. Thanks for clarifying!